src/main/kotlin/app/revanced/patches/twitch/misc/settings/fingerprints/MenuGroupsUpdatedFingerprint.kt

```diff
@@ -0,0 +1,16 @@
+package app.revanced.patches.twitch.misc.settings.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patches.twitch.misc.settings.annotations.SettingsCompatibility
+
+@Name("settings-menu-groups-update-fingerprint")
+@SettingsCompatibility
+@Version("0.0.1")
```

Comment:
Annotations for fingerprints have been deprecated a while back, you can safely remove them.